tests/test_response.py

```diff
@@ -122,6 +123,27 @@ class RandomException(LambdaFWException):
             "statusCode": 500,
         }
 
+    def test__from_exception__adds_extra_data_to_response_body_if_only_declared(self) -> None:
+        class RandomException(LambdaFWException):
+            error_code = "RAND001"
+
+        response = Response.from_exception(
+            RandomException(extra={"added_data": "random"}), "req-id"
+        )
+
+        assert response.to_dict() == {
+            "body": ANY,
+            "headers": {"Content-Type": ContentType.JSON},
+            "isBase64Encoded": False,
+            "statusCode": 500,
+        }
+        assert response.body == {
```

Comment:
Hmmm, if you didn't plan to use value from `.to_dict()`, you can compare all the attributes this way 😉